crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -108,3 +108,15 @@ struct ClusterOffsetsAndCounts {
     data: array<vec4<u32>, 1024u>,
 };
 #endif
+
+struct ReflectionProbe {
+    inverse_transform: mat4x4<f32>,
```

Comment:
Can maybe be packed with affine to square or whatever to save some space? Idk

Reply:
I switched this to the transpose to save space.